lib/saxerator/latches/child_of.rb

```diff
@@ -31,7 +31,7 @@ def increment_depth(amount)
       end
 
       def depth_within_element
-        @depths.size > 0 ? @depths.last : 0
+        @depths.length > 0 ? @depths[-1] : 0
```

Comment:
I did a benchmark and `size` and `length` have the same performance.

The `-1` index is slightly faster than `.last` though